### DISCUSSION TITLE
Instead of editing message how to reply in `inlinekeyboard` examples?

### DISCUSSION THREAD

**Discussion by refl3ct1on** [python-telegram-bot/python-telegram-bot#3721] [Q&A]
I've check both inline keyboard example provided in the example folder and I could not find a way to modify them to send new text message.

First thing that I tried is changing `query.edit_message_text`to `update.message.reply_text` but it throws an error:

```python
Traceback (most recent call last):
  File ".venv/lib/python3.11/site-packages/telegram/ext/_application.py", line 1124, in process_update
    await coroutine
  File "venv/lib/python3.11/site-packages/telegram/ext/_handler.py", line 141, in handle_update
    return await self.callback(update, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "bot.py", line 57, in button
    await update.message.reply_text(text=f"Selected option: {query.data}")
          ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'reply_text'
```

is there a way to keep the old messages (sending new one, instead of editing it)  in the chat history so that user can see them?

**Comment by lemontree210** [comment]
Hi! Thanks for reaching out. You can `await update.effective_chat.send_message()`.

[`update.effective_chat`](https://docs.python-telegram-bot.org/en/stable/telegram.update.html#telegram.Update.effective_chat) gives you a `Chat` object that has method [`.send_message()`](https://docs.python-telegram-bot.org/en/stable/telegram.chat.html#telegram.Chat.send_message)

You can also use method `.reply_[text/html/markdown_v2]` of [`update.effective_message`](https://docs.python-telegram-bot.org/en/stable/telegram.update.html#telegram.Update.effective_message)
